app/src/main/java/org/covidwatch/android/ui/menu/MenuFragment.kt

```diff
@@ -44,4 +57,34 @@ class MenuFragment : Fragment(R.layout.fragment_menu) {
         val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
         startActivity(browserIntent)
     }
-}
\ No newline at end of file
+
+    private fun makeTestExposureNotification() {
+        val exposureInformationRepository: ExposureInformationRepository by inject()
```

Comment:
You will not need this reference there :)

Reply:
I need it because straightforward initialization requires a parameter in the constructor.